HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -4972,22 +4988,23 @@ def initialize
 
 # Object with sizing values (loads, capacities, airflows, etc.) for a specific HVAC system
 class HVACSizingValues
-  attr_accessor(:Cool_Load_Sens,      # [Double] Total sensible cooling load (Btu/hr)
-                :Cool_Load_Lat,       # [Double] Total latent cooling load (Btu/hr)
-                :Cool_Load_Tot,       # [Double] Total (sensible + latent) cooling load (Btu/hr)
-                :Heat_Load,           # [Double] Total heating sensible load (Btu/hr)
-                :Heat_Load_Supp,      # [Double] Total heating sensible load for the HP backup (Btu/hr)
-                :Cool_Capacity,       # [Double] Nominal total cooling capacity (Btu/hr)
-                :Cool_Capacity_Sens,  # [Double] Nominal sensible cooling capacity (Btu/hr)
-                :Heat_Capacity,       # [Double] Nominal heating capacity (Btu/hr)
-                :Heat_Capacity_Supp,  # [Double] Nominal heating capacity for the HP backup (Btu/hr)
-                :Cool_Airflow,        # [Double] Cooling airflow rate (cfm)
-                :Heat_Airflow,        # [Double] Heating airflow rate (cfm)
-                :GSHP_Loop_Flow,      # [Double] Ground-source heat pump water flow rate through the geothermal loop (gal/min)
-                :GSHP_Bore_Holes,     # [Integer] Ground-source heat pump number of boreholes (#)
-                :GSHP_Bore_Depth,     # [Double] Ground-source heat pump depth of each borehole (ft)
-                :GSHP_G_Functions,    # [Array<Array<Double>, Array<Double>>] Ground-source heat pump G-functions
-                :GSHP_Bore_Config)    # [String] Ground-source heat pump borefield configuration (HPXML::GeothermalLoopBorefieldConfigurationXXX)
+  attr_accessor(:Cool_Load_Sens,      # Total sensible cooling load (Btu/hr)
```

Comment:
When you resolved conflicts, you reverted a bunch of changes here and below that we want. Can you fix this?

Reply:
I see the diffs, they were just in the comments around the HVACSizingValues class. I resolved them to match master and committed.